roles/import_image_and_run_gcp_instance/tasks/main.yml

```
@@ -0,0 +1,113 @@
+---
+- name: Run 'cloud.gcp_ops.import_image_and_run_gcp_instance' role
+  module_defaults:
+    group/gcp: "{{ gcp_setup_credentials__output }}"
+
+  block:
+    - name: Fail when 'import_image_and_run_gcp_instance_bucket_name' is undefined
+      ansible.builtin.fail:
+        msg: Storage bucket name to host the .raw image must be defined as import_image_and_run_gcp_instance_bucket_name
+      when: import_image_and_run_gcp_instance_bucket_name is undefined
+
+    - name: Fail when 'import_image_and_run_gcp_instance_image_path' is undefined
+      ansible.builtin.fail:
+        msg: The .raw image path must be defined as import_image_and_run_gcp_instance_image_path
+      when: import_image_and_run_gcp_instance_image_path is undefined
+
+    - name: Fail when 'import_image_and_run_gcp_instance_instance_name' is undefined
+      ansible.builtin.fail:
+        msg: The name to assign to the GCP instance must be defined as import_image_and_run_gcp_instance_instance_name
+      when: import_image_and_run_gcp_instance_instance_name is undefined
+
+    - name: Fail when 'import_image_and_run_gcp_instance_import_image_name' is undefined
+      ansible.builtin.fail:
+        msg: The name of the GCP custom image must be defined as import_image_and_run_gcp_instance_import_image_name
+      when: import_image_and_run_gcp_instance_import_image_name is undefined
+
+    - name: Check if an instance with the specified name alrerady exists
+      google.cloud.gcp_compute_instance_info:
+        zone: "{{ import_image_and_run_gcp_instance_zone }}"
```

```suggestion
        zone: "{{ import_image_and_run_gcp_zone }}"
```
It looks like this var drops `instance` everywhere else.